src/models/pang15/repairs.md

```diff
@@ -0,0 +1,242 @@
+
+## Replacing the cooling system:
+
+The Pangolin 15 has a fan and heatsink assembly to cool the CPU. The fans and heatsink are held together with adhesive; it is possible to replace the fans individually, but removing the entire assembly is recommended to perform service.
+
+If the fan becomes noisy and cleaning it out doesn't fix the issue, you may need a new fan. [Contact support](https://support.system76.com/) to start a warranty claim or parts purchase.
+
+Depending on your climate and the age of the machine, it may be necessary to apply new thermal paste between the CPU and the heatsink. Thermal paste helps facilitate effective heat transfer between the CPU and the cooling equipment.
+
+**Part numbers:**
+- The heatsink's part number is `01.WW7.APX938ARM00`.
+- The left fan is a Joray `BF070E12P (S-0592)`, and its part number is `01.SS9.APX938DTH00`.
+- The right fan is a Joray `BF080D12P (S-0593)`, and its part number is `01.SS9.APX938DTH01`.
+
+**Tools required:** Cross-head (Phillips) screwdriver, thermal paste  
+**Time estimate:** 20 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+### Steps to replace the fan/heatsink/thermal paste:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Slowly peel off the black coverings and their connecting strip from the heatsink, highlighted green below.
+
+![Thermal system covers](./img/thermal-system-covers.webp)
+
+3. Remove the six fan screws, highlighted green below.
+
+![Thermal screws](./img/thermal-system.webp)
+
+4. Unplug the fan connectors, highlighted cyan above.
+5. Unscrew the four heatsink screws, highlighted red above.
+    - The heatsink screws are held captive and will not come out of the heatsink.
+6. Remove the fan and heatsink assembly from the system, being careful not to bend the heatsink pipe. It may take some pressure to break the seal of the thermal paste.
+    - The fans are held onto the heatsink with adhesive strips on both the top and bottom; if they come loose, place them back into position between the strips before reinstalling the fan/heatsink assembly.
+7. Using a paper towel, remove the existing silver thermal paste. You may also use a small amount of rubbing alcohol if the old paste is dried or difficult to remove.
+    - The pink thermal pads do not need to be removed.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+8. After cleaning the CPU and heatsink, apply a small line of thermal paste directly onto the CPU chip.
+
+![Thermal paste application](./img/thermal-paste-application.webp)
+
+9. Carefully replace the heatsink.
+10. Tighten the four heatsink screws.
+11. Replace the fan screws, plug in the fan connectors, stick the black coverings back onto the heatsink pipes, and replace the bottom panel.
+
+## Replacing the speakers:
+
+The system has four bottom-firing speakers (two tweeters and two woofers). The left and right channels can be replaced independently.
```

Comment:
The two little tweeter ones are top-firing

Reply:
Makes me wonder why the woofers are bottom-firing... updated the note.